### PR TITLE
DONT MERGE: test reviewdog

### PR DESCRIPTION
**Pull request by Vincent-lau**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -179,6 +179,7 @@ jobs:
           github_token: ${{ secrets.github_token }}
           reporter: github-pr-review
           level: error
+          diff: git diff ${{ github.base_ref }} -- . :^ocaml/xapi-idl/lib_test/test_data
```
Comment (psafont) [inline .github/workflows/main.yml:182]: tempted to just do this:
```
          diff: git diff ${{ github.base_ref }} -- . :^ocaml
```

Comment (Vincent-lau) [reply .github/workflows/main.yml:182]: yeah does not really work 
```
Warning: Unexpected input(s) 'diff', valid inputs are ['entryPoint', 'args', 'github_token', 'level', 'reporter']

```

**Comment by Vincent-lau** [comment]
seems like there is no easy workaround for this one. The reviewdog/pyflakes action does not support customising the input files to be reviewed....

**Comment by psafont** [comment]
I'll work on this on the afternoon

**Comment by Vincent-lau** [comment]
Resolved by @psafont in #5575, closing this PR

**Pull request closed without merging**